chainerio/__init__.py

```diff
@@ -68,60 +87,194 @@ def open(file_path: str, mode: str = 'rb',
         errors, newline, closefd, opener)
 
 
-def set_root(uri_or_handler: str) -> None:
+def set_root(uri_or_handler: Union[str, Type['IO']]) -> None:
+    """Sets the current context to ``uri_or_handler``
+
+    The context here refers to the default handler and the root path.
+    The default handler points to a filesystem or a container which
+    ChainerIO uses when called without a full URI. Overrides by the full URI.
```

Comment:
I would prefer "absolute URI" than full, c.f. https://www.w3.org/Addressing/URL/4_3_Partial.html

Reply:
"Overrides by the full URI." is this sentence a residue?

Reply:
I use only "URI", as in the URI is defined with scheme

Reply:
I meant the term "full" was unnecessary, or could be replaced by "absolute" regarding the reference I mentioned above.